src/elements.js

```diff
@@ -220,6 +220,39 @@ _.register({
 		subtree: true
 	},
 
+	"select[multiple]": {
+		extend: "select",
+		selector: "select[multiple]",
+		getValue: element => {
+			const selected = element.selectedOptions;
+			const value = [];
+
+			if (selected.length) {
+				for (const option of selected) {
+					value.push(option.value);
+				}
+			}
+
+			return value;
+		},
+		setValue: (element, value) => {
+			value = Array.isArray(value)? value : [value];
+
+			$$("option", element).forEach(option => {
```

Comment:
Slightly preferable to use `element.options` instead of a new DOM query.

Reply:
You are absolutely right!
I must examine the DOM API more thoughtfully. 😞